pkg/kube/pod_command_executor.go

```diff
@@ -21,6 +21,41 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type ExecError interface {
+	error
+	Stdout() string
+	Stderr() string
+}
+
+type execError struct {
+	cause  error
```

Comment:
if you embed it, you will be able to get rid of `Error()` method.

Reply:
We had a discussion and figured out that interface is not needed here, so I kept only struct and made it exported.